### ISSUE TITLE
visitconvert in parallel can't launch due to "-plugindir" and other arguments

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#2188]
The internallauncher script adds "-plugindir", "-visithome", and "-visitarchhome" to the command line in, for example, JobSubmitter_mpirun (and others).  The engine knows what to do with these, but other binaries don't.  (Like visitconvert.)

This wouldn't necessarily be a problem, because visitconvert will ignore flags it doens't understand, so assuming the environment variables still work, visitconvert would be happy.

The *real* problem is that JobSubmitter_mpirun (and others) add these arguments before the other args, and visitconvert expects its first argument to be the input file name.  (I.e. it assumes options should go at the end of the command line, not the beginning.)

I don't know if there's any priority in making visitconvert (and other binaries) understand the -plugindir (and similar) flags, and I suspect we don't want to bother making visitconvert parse its input file differently.

This means the easiest solution is to either force those -plugindir (and similar) flags to the end of the command line, or simply to not add them if it's not the engine we're launching.

(I didn't try running 2.6.2 to see if any of this was fixed, but the important pieces of source code are the same as 2.6.1, so I assume it's still a problem.)

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Jeremy Meredith
Original creation: 03/19/2013 02:38 pm
Original update: 03/26/2013 01:24 pm
Ticket number: 1388
